llvm/lib/Target/X86/X86FastISel.cpp

```diff
@@ -2588,6 +2588,15 @@ bool X86FastISel::TryEmitSmallMemcpy(X86AddressMode DestAM,
   return true;
 }
 
+// Add an annotation to an intrinsic instruction, specifying whether the
+// intrinsic has been inlined or not.
+void annotateIntrinsic(const IntrinsicInst *II, const char *S) {
```

Comment:
I think this weird `S` thing should be a `bool` so we don't have to percolate knowledge about the valid contents around the whole system.

Reply:
Fixed in e9b3a75.